src/store-api/src/metadata.rs

```diff
@@ -134,6 +135,19 @@ pub struct RegionMetadata {
     pub schema_version: u64,
 }
 
+impl fmt::Debug for RegionMetadata {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        f.debug_struct("RegionMetadata")
+            .field("column_metadatas", &self.column_metadatas)
```

Comment:
We'd also customize the debug output of `ColumnMetadata`.

For each column, we can print it like:
```
ColumnMetadata[<name>  <type> <nullability> <semantic type> <deafault constraints>  <metadata> <id>]
```

For example, for a column that:
- name is "ts"
- type is `TimestampMillisecond`
- non-null
- semantic type is `TIMESTAMP`
- default to current timestamp
- no metadata
- id is `1`

Then the output should be like:
```
ColumnMetadata[ts TimestampMillisecond not null TIMESTAMP default=Some(Function("current_timestamp")) 1] 
```

> We skip `metadata` field since this column does not have any.